src/Illuminate/Collections/Arr.php

```diff
@@ -642,9 +642,10 @@ public static function mapWithKeys(array $array, callable $callback)
     /**
      * Run a map over each nested chunk of items.
      *
+     * @template TKey
      * @template TMapSpreadValue
```

Comment:
This can also be renamed for consistency---method templates are scoped to the method, so there's no need to complicate it by adding the name of the method to the template name

```suggestion
     * @template TValue
```